module1.js

```diff
@@ -14,10 +14,9 @@
 */
 
 function question1(name1) {
-  if (name1) {
-    // TODO:
-    return;
-  }
+  if (name1 === "Code Your Future") {
```

Comment:
This does the job, but can you think of a way to write this implementation on just one line?